src/bobbit/modules/duckhunt.py

```diff
@@ -36,21 +36,27 @@
     'WHOOSH! You missed the duck completely!',
     'Your gun jammed!',
     'Better luck next time.',
-    'WTF!? Who are you Dick Cheney?'
+    'WTF!? Who are you, Dick Cheney?'
 )
 SAVE_MISSES = (
     "The duck didn't want to be friends, maybe next time.",
     "Well this is awkward, the duck needs to think about it.",
     "The duck said no, maybe bribe it with some pizza? Ducks love pizza don't they?",
     "Who knew ducks could be so picky?"
 )
+COOLDOWN_AMOUNT = 7.0
+COOLDOWN_MESSAGE = f" You can try again in {COOLDOWN_AMOUNT} seconds."
```

Comment:
I'd prefer single quotes

Reply:
You don't use single quotes elsewhere, but I can change them.